### DISCUSSION TITLE
Network error while installing brew

### DISCUSSION THREAD

**Discussion by Byeongju-Kong** [Homebrew/discussions#1078] [Getting started]
I run into network error. while installing brew, I turn network of Wi-Fi into mobile hotspot. 
What should I do?
<img width="570" alt="스크린샷 2021-03-19 14 11 06" src="https://user-images.githubusercontent.com/68771917/111734222-05a79f80-88bd-11eb-8ba7-d981a2f8f094.png">

**Comment by SMillerDev** [comment]
That's not a network isue, it's a permission issue. You need to change the ownership of these directories to no longer be world writable. You can use `chmod -R 600 /directory/path/here` to do that.

**Comment by Byeongju-Kong** [comment]
I got it. Thank U!
